src/helper/task.ts

```diff
@@ -735,6 +737,20 @@ export class ZkWasmServiceHelper {
     return config;
   }
 
+  async queryProverNodeTimerangeStats(
```

Comment:
For such query, in fact, we may do not want user to compose their own ProverNodeTimerangeStatsParams, we'd better just provide a API which inputs are prover_address, start ts, end ts. We can compose the query param internal.

Also, for the API, ask user to input string as the timestamp is not good. If we want to pass string to  JSON, we can do the converting internally in the API function, but the API params should be some datetime type.